src/components/Message/Message.stories.tsx

```diff
@@ -0,0 +1,147 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+import { useState } from 'react'
+
+import { Hierarchy, Size } from '../Button/Button.types'
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import { Message } from '.'
+import { useFeedbackMessage } from '../../hooks/useFeedbackMessage/useFeedbackMessage'
+
+const meta = {
+  args: {
+    message: 'This is a Feedback message',
+  },
+  component: Message,
+} satisfies Meta<typeof Message>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const FeedbackMessage: Story = {
+  decorators: [(_Story, context) => {
+    const { info, loading, success, warning, error } = useFeedbackMessage()
+
+    const { message } = context.args
+
+    return (
+      <div style={{ display: 'flex', gap: '4px' }}>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="blue" name="PiInfo" size={16} />}
+          onClick={() => info({ message })}
+        >
+Info
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="green" name="PiCheck" size={16} />}
+          onClick={() => success({ message })}
+        >
+Success
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon name="PiSpinner" size={16} />}
+          onClick={() => loading({ message })}
+        >
+Loading
+        </Button>
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="orange" name="PiWarning" size={16} />}
+          onClick={() => warning({ message })}
+        >
+Warning
+        </Button>
+        {/* Use icons in btn */}
+        <Button
+          hierarchy={Hierarchy.Neutral}
+          icon={<Icon color="red" name="PiStop" size={16} />}
+          onClick={() => error({ message })}
+        >
+Error
+        </Button>
+      </div >
+    )
+  }],
+}
+
+export const FeedbackMessageWithExtraContent: Story = {
+  decorators: [() => {
+    const { success, dismiss } = useFeedbackMessage()
+
+    const onDismiss = (): void => { dismiss('messageKey') }
+
+    const onClick = (): void => {
+      success({
+        duration: 0,
+        extra: <Button size={Size.Small} onClick={onDismiss}>Close</Button>,
+        key: 'messageKey',
+        message: 'This is a feedback message',
+      })
+    }
+
+    return (
+      <div>
+        <Button onClick={onClick}>Click me to show a Feedback Message</Button>
+      </div >
+    )
+  }],
+}
```

Comment:
The button used inside the Feedback Message is the same of the Design System. 

The reason why the color is different is because the theme is not imported in the Storybook's story since we generate the component with the hook.